packages/reports/src/Containers/statement.tsx

```diff
@@ -247,7 +248,12 @@ const Statement = observer(({ component_icon }: TStatement) => {
 
     return (
         <React.Fragment>
-            <ReportsMeta className='reports__meta--statement' filter_component={<FilterComponent />} is_statement />
+            <ReportsMeta
+                className='reports__meta--statement'
+                filter_component={<FilterComponent />}
+                is_statement
+                key={current_language}
```

Comment:
`ReportsMeta` component was not rerendering on language change. Force it to do it by adding key